homeassistant/components/reolink/number.py

```diff
@@ -493,6 +506,165 @@ class ReolinkChimeNumberEntityDescription(
+        method=lambda api, ch, loc, value: api.baichuan.set_smart_ai(
+            ch, "loss", loc, sensitivity=int(value)
+        ),
+    ),
+    ReolinkSmartAINumberEntityDescription(
+        key="intrusion_delay",
+        smart_type="intrusion",
+        cmd_id=529,
+        translation_key="intrusion_delay",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        native_min_value=0,
+        native_max_value=10,
+        supported=lambda api, ch: api.supported(ch, "ai_intrusion"),
+        value=lambda api, ch, loc: api.baichuan.smart_ai_delay(ch, "intrusion", loc),
+        method=lambda api, ch, loc, value: api.baichuan.set_smart_ai(
+            ch, "intrusion", loc, delay=int(value)
+        ),
+    ),
+    ReolinkSmartAINumberEntityDescription(
+        key="linger_delay",
+        smart_type="loitering",
+        cmd_id=531,
+        translation_key="linger_delay",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        native_min_value=1,
+        native_max_value=10,
+        supported=lambda api, ch: api.supported(ch, "ai_linger"),
+        value=lambda api, ch, loc: api.baichuan.smart_ai_delay(ch, "loitering", loc),
+        method=lambda api, ch, loc, value: api.baichuan.set_smart_ai(
+            ch, "loitering", loc, delay=int(value)
+        ),
+    ),
+    ReolinkSmartAINumberEntityDescription(
+        key="forgotten_item_delay",
+        smart_type="legacy",
+        cmd_id=549,
+        translation_key="forgotten_item_delay",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        native_min_value=1,
+        native_max_value=30,
+        supported=lambda api, ch: api.supported(ch, "ai_forgotten_item"),
+        value=lambda api, ch, loc: api.baichuan.smart_ai_delay(ch, "legacy", loc),
+        method=lambda api, ch, loc, value: api.baichuan.set_smart_ai(
+            ch, "legacy", loc, delay=int(value)
+        ),
+    ),
+    ReolinkSmartAINumberEntityDescription(
+        key="taken_item_delay",
+        smart_type="loss",
+        cmd_id=551,
+        translation_key="taken_item_delay",
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.CONFIG,
+        native_step=1,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        native_min_value=1,
+        native_max_value=30,
+        supported=lambda api, ch: api.supported(ch, "ai_taken_item"),
+        value=lambda api, ch, loc: api.baichuan.smart_ai_delay(ch, "loss", loc),
+        method=lambda api, ch, loc, value: api.baichuan.set_smart_ai(
+            ch, "loss", loc, delay=int(value)
+        ),
+    ),
```

Comment:
No device class?

Reply:
I don't think there is any fitting device class for detection sensitivity or detection delay?
I looked in this list: https://developers.home-assistant.io/docs/core/entity/number/#available-device-classes

Reply:
duration?

Reply:
Somehow I looked over that one.
I added it to all fitting entities withing Reolink number entities.